daemon/src/projection.rs

```diff
@@ -487,7 +487,10 @@ impl Cfd {
 
                 self.state = CfdState::PendingCommit;
             }
-            RevokeConfirmed => todo!("Deal with revoked"),
+            RevokeConfirmed => {
+                tracing::error!(order_id = %self.order_id, "Revoked logic not implemented");
+                self.state = CfdState::PendingCet;
```

Comment:
For `CommitConfirmed`, we set this to `OpenCommitted`.

`PendingCet` is only meant for when we actually published the CET and it is pending in mempool.

Reply:
Ah ok. I;ll change it.